### PR TITLE
Adding Courses in English

### PR DESCRIPTION
**Pull request by Shashwat79802**
## What does this PR do?
The PR add's several courses in English language

### Why is this valuable (or not)?
The courses are of high knowledge quality from expert YouTubers and also from Google

### How do we know it's really free?
These are absolutely free because these are mostly from YouTube and one is from Google

## Checklist:
- [☑️] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [☑️] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [☑️] Include author(s) and platform where appropriate.
- [☑️] Put lists in alphabetical order, correct spacing.
- [☑️] Add needed indications (PDF, access notes, under construction).
- [☑️] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
please resolve conflicts.
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes
Don't invent titles! Use the title given by the resource.

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by Shashwat79802** [comment]
"YouTube videos. We usually don't link to individual YouTube videos unless they are more than an hour long and are structured like a course or a tutorial." - from platforms-and-access-notes. Her video is a 5hour long video, and is the best as I myself had learned Python from her.
And can you please briefly describe the conflict, I'm actually not able to get it what change you want with the PR.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -520,6 +523,7 @@
 * [HTML and CSS Crash For Beginner](https://youtube.com/playlist?list=PL4-IK0AVhVjM0xE0K2uZRvsM7LkIhsPT-) - Kevin Powell (YouTube)
 * [HTML and CSS Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhwNC5ppZo_dYGVjerQY3xYU) - Dani Krossing
 * [HTML Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWr_FB-hVEgEGUESZL1TOiJ6)
+* [HTML Tutorial for Beginners](https://www.youtube.com/watch?v=qHB2jUvAlGo) - Programming with Harry
```
Comment (davorpa) [inline courses/free-courses-en.md:526]:
```suggestion
* [HTML Tutorial For Beginners (With Notes)](https://www.youtube.com/watch?v=qHB2jUvAlGo) - ProgrammingWithHarry
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -923,6 +929,7 @@
 * [Python Programming Tutorials](https://www.youtube.com/playlist?list=PLzMcBGfZo4-mFu00qxl0a67RhjjZj3jXm) - Tech With Tim
 * [Python Tutorial - Python for Beginners [Full Course]](https://www.youtube.com/watch?v=_uQrJ0TkZlc) - Moshfegh Hamedani (Programming with Mosh)
 * [Python tutorial for beginners](https://www.youtube.com/playlist?list=PLK8cqdr55Tss0puRoHDBagvj7Qjin9axl) - ProgrammingWithHarry
+* [Python Tutorial for Beginners](https://www.youtube.com/watch?v=t8pPdKYpowI) - Nana Janashia (TechWorld with Nana)
```
Comment (davorpa) [inline courses/free-courses-en.md:932]:
```suggestion
* [Python Tutorial for Beginners - Learn Python in 5 Hours \[FULL COURSE\]](https://www.youtube.com/watch?v=t8pPdKYpowI) - Nana Janashia (TechWorld with Nana)
```

**Comment by Shashwat79802** [comment]
Thanks for helping me out.

**Comment by eshellman** [comment]
closing and reopening because the linter seems to have not run.

**Comment by eshellman** [comment]
@davorpa Any idea why this PR has lost its actions?

**Comment by davorpa** [comment]
> ~@davorpa Any idea why this PR has lost its actions?~ Never mind.

No, sorry. Sometimes happens if too much runs in a run are executed. Are losed / queued without can be seen in the UI

**Review by davorpa — APPROVED** [review]

**Comment by Shashwat79802** [comment]
Do I have to do anything else?

**Pull request merged**